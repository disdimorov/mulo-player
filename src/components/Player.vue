<template>
  <div class="player">player</div>
</template>

<script>
// TODO add unit
export default {
  name: 'Player',
};
</script>

<style scoped lang="less">
.player {}
</style>
