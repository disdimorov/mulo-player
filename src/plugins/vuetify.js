import Vue from 'vue';
import Vuetify from 'vuetify';
import { THEME as theme } from '@/const';

Vue.use(Vuetify);

export default new Vuetify({ theme });
