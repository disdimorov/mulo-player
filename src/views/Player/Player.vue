<template>
  <div class="player">player</div>
</template>

<script>
export default {
  name: 'Player',
};
</script>

<style scoped lang="less">
.player {}
</style>
