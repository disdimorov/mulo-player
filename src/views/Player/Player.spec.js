import { mount, shallowMount, createLocalVue } from '@vue/test-utils';
import vuetify from '@/plugins/vuetify';
import Player from './Player.vue';

const localVue = createLocalVue();

describe('Component Player', () => {
  const createWrapper = ({ isWithoutStubs = false, ...options } = {}) => {
    const renderer = isWithoutStubs ? mount : shallowMount;
    const wrapper = renderer(Player, {
      localVue,
      vuetify,
      ...options,
    });

    return { wrapper };
  };

  describe('snapshots', () => {
    it('should match snapshot', () => {
      const { wrapper } = createWrapper({ isWithoutStubs: true });

      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
