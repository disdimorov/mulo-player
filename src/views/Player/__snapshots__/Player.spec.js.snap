// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Player snapshots should match snapshot 1`] = `<div class="player">player</div>`;
